api/src/main/java/com/okta/idx/sdk/api/client/IDXClient.java

```diff
@@ -53,6 +54,8 @@
 
     IDXResponse recover(RecoverRequest recoverRequest, String href) throws ProcessingException;
 
+    IDXResponse poll(PollRequest pollRequest, String href) throws ProcessingException;
```

Comment:
This is another breaking API change, right? I'm wondering if this is the right public abstraction. Since it's caused us to pump the version number twice now.

Reply:
Introducing a new API to an interface isn't actually considered to be a breaking binary change (surprisingly).